aws/resource_aws_cloudfront_distribution.go

```diff
@@ -22,7 +22,7 @@ func resourceAwsCloudFrontDistribution() *schema.Resource {
 		Update: resourceAwsCloudFrontDistributionUpdate,
 		Delete: resourceAwsCloudFrontDistributionDelete,
 		Importer: &schema.ResourceImporter{
-			State: resourceAwsCloudFrontDistributionImport,
+			State: schema.ImportStatePassthrough,
```

Comment:
The two `d.Set()` calls in the previous import function were actually to help practitioners not show a difference on the first apply after import if they are using the default values of "virtual"/Terraform-only attributes in their configuration:

```go
	// This is a non API attribute
	// We are merely setting this to the same value as the Default setting in the schema
	d.Set("retain_on_delete", false)
	d.Set("wait_for_deployment", true)
```

I'd recommend keeping them in there:

```suggestion
			State: func(d *schema.ResourceData, meta interface{}) ([]*schema.ResourceData, error) {
				d.Set("retain_on_delete", false)
				d.Set("wait_for_deployment", true)
				return []*schema.ResourceData{d}, nil
			},
```

It looks like this guidance about "virtual" attributes is not present in the [Extending Terraform documentation](https://www.terraform.io/docs/extend/resources/import.html), so I will take that as an action item here.

Reply:
ahh I read the comment thinking it sounded redundant but I forgot the defaults don't get filled in at import time, will fix!